### ISSUE TITLE
aws-android-sdk-lex is not sending requestAttributes

### ISSUE THREAD

**Issue by xiezhengfeng** [aws-amplify/aws-sdk-android#801]
**Describe the bug**
When I use aws-android-sdk-lex to send a request to my Lex bot, I found that the requestAttributes was not sending.

**To Reproduce**
A code sample or steps:
```
    private void textEntered() {
        String text = userTextInput.getText().toString();
        if (!inConversation) {
            Log.d(TAG, " -- New conversation started");
            startNewConversation();
            addMessage(new TextMessage(text, "tx", getCurrentTimeStamp()));
            Map<String, String> requestAttributes = new HashMap<>();
            requestAttributes.put("foo", "bar");
            lexInteractionClient.textInForTextOut(text, null, requestAttributes);
            inConversation = true;
        } else {
            Log.d(TAG, " -- Responding with text: " + text);
            addMessage(new TextMessage(text, "tx", getCurrentTimeStamp()));
            Map<String, String> requestAttributes = new HashMap<>();
            requestAttributes.put("foo", "bar");
            convContinuation.continueWithTextInForTextOut(text, requestAttributes);
        }
        clearTextInput();
    }
```

**Which AWS service(s) are affected?**
Lex
**Expected behavior**
Lex bot receives the requestAttributes: {"foo": "bar"}

**Screenshots**
![image](https://user-images.githubusercontent.com/7782120/54571286-73f8b500-49b8-11e9-8590-c40e67591dda.png)

**Environment Information (please complete the following information):**
 - AWS Android SDK Version: 2.12.5
 - Device: Oneplus 6
 - Android Version: Pie 9.0.4
 - Specific to simulators: No

**Additional context**
I believe the issue is in this the method generateRequestInternal() in com.amazonaws.mobileconnectors.lex.interactionkit.utils.CreateLexServiceRequest.java. Which has an input parameter of requestAttributes but do nothing about that.
```
private static PostContentRequest generateRequestInternal(Map<String, String> sessionAttributes,
                                                              Map<String, String> requestAttributes,
                                                              InteractionConfig interactionConfig,
                                                              AWSCredentialsProvider credentialsProvider,
                                                              ResponseType mode) {
        final PostContentRequest request = new PostContentRequest();

        request.setBotName(interactionConfig.getBotName());
        request.setBotAlias(interactionConfig.getBotAlias());
        request.setAccept(mode.toString());

        final Map<String, String> newSessionAttributes = new HashMap<String, String>();
        newSessionAttributes.putAll(interactionConfig.getGlobalSessionAttributes());
        if (sessionAttributes != null) {
            newSessionAttributes.putAll(sessionAttributes);
        }

        request.setSessionAttributes(mapToBase64(newSessionAttributes));

        if (interactionConfig.getUserId() == null || interactionConfig.getUserId().isEmpty()) {
            final CognitoCredentialsProvider cognitoCredentialsProvider = (CognitoCredentialsProvider) credentialsProvider;
            request.setUserId(cognitoCredentialsProvider.getIdentityId());
        } else {
            request.setUserId(interactionConfig.getUserId());
        }

        return request;
    }
```

**Comment by minbi** [comment]
Hi @xiezhengfeng ,

The interaction client currently does not support this feature of the service. If are able to pinpoint a solution before we are able to address this issue. We are open to pull requests.

**Comment by desokroshan** [comment]
@xiezhengfeng  This has been fixed in the latest release [2.12.7](https://github.com/aws-amplify/aws-sdk-android/releases/tag/release_v2.12.7). Could you please update to the latest version and confirm. Thanks!
